lib/FinderNavigation/styles.scss

```diff
@@ -26,12 +26,13 @@ $finder-indent-level-support: 10 !default;
 }
 
 .finder__item-content {
+  padding-right: $layout-spacing-base/2;
   display: flex;
   cursor: default;
   height: 35px;
   line-height: 35px;
   transition: height 200ms;
-
+  padding-left: $layout-spacing-base/2;
```

Comment:
One tiny thing, maybe we could do:

`padding: 0 $layout-spacing-base/2;`

Reply:
I was about to say "no! I only want it on the left!", then I realised at the start of this block I have the exact same thing for padding right. So yes absolutely, nice catch 😅